lib/src/package_meta.dart

```diff
@@ -157,31 +155,28 @@ abstract class PubPackageMeta extends PackageMeta {
   PubPackageMeta(Folder dir, ResourceProvider resourceProvider)
       : super(dir, resourceProvider);
 
-  static List<List<String>> __sdkDirFilePaths;
-
-  static List<List<String>> get _sdkDirFilePaths {
-    if (__sdkDirFilePaths == null) {
-      __sdkDirFilePaths = [];
-      if (Platform.isWindows) {
-        for (var paths in __sdkDirFilePathsPosix) {
-          var windowsPaths = [
-            for (var path in paths)
-              p.joinAll(p.Context(style: p.Style.posix).split(path)),
-          ];
-          __sdkDirFilePaths.add(windowsPaths);
-        }
-      } else {
-        __sdkDirFilePaths = __sdkDirFilePathsPosix;
+  static late final List<List<String>> _sdkDirFilePaths = () {
+    var __sdkDirFilePaths = <List<String>>[];
```

Comment:
Nit: Don't use `_` for local variables. https://dart.dev/guides/language/effective-dart/style#dont-use-a-leading-underscore-for-identifiers-that-arent-private

But it's on existing code, so feel free to disregard.

Reply:
done